program/src/instructions/compressed/list.rs

```diff
@@ -116,13 +121,17 @@ pub fn process_list<'info>(
         Some(expire_in_sec) => {
             let expire_in_i64 = i64::try_from(expire_in_sec).unwrap();
             require!(expire_in_i64 <= MAX_EXPIRY_SEC, TcompError::ExpiryTooLarge);
+            // TODO: no checked math?
             Clock::get()?.unix_timestamp + expire_in_i64
         }
+        // TODO: no checked math?
         None => Clock::get()?.unix_timestamp + MAX_EXPIRY_SEC,
     };
     list_state.expiry = expiry;
     list_state.rent_payer = ctx.accounts.rent_payer.key();
 
+    // TODO: I wonder if this opens up room for error? The person creating the listing adds a cosigner, but forgets to make them a signer. Now they think it's stored, but it's not. 
+    // We arleady have the Option<> so there is a way for someone to not pass it. I wonder if we should hard error if a cosigner is passed but not a signer?
```

Comment:
The account could be a proof account which is `Some(account)` but not a signer so we have to handle that case